src/app/pages/HomePage/Graph/ParaTimeSelector/index.tsx

```diff
@@ -161,7 +166,9 @@ const ParaTimeSelectorCmp: FC<ParaTimeSelectorProps> = ({ disabled, step, setSte
   }, [isMobile, step, setStep])
 
   const onExploreClick = () => {
-    if (isMobile) {
+    const mobileHelpScreenShown = localStore.get(StorageKeys.MobileHelpScreenShown)
+
+    if (isMobile && !mobileHelpScreenShown) {
```

Comment:
It still breaks after:
- user clicks Explore Oasis
- goes through the help
- clicks Toggle info screen

or
- refreshes
- clicks Toggle info screen
- goes through the help
- clicks Toggle info screen

Reply:
Resolved now, minor bug. Thanks for noticing yet again